src/components/NewLexemeForm.vue

```diff
@@ -4,7 +4,11 @@ import { useStore } from 'vuex';
 import { Button as WikitButton } from '@wmde/wikit-vue-components';
 import { useMessages } from '@/plugins/MessagesPlugin/Messages';
 import LemmaInput from '@/components/LemmaInput.vue';
-import { UPDATE_LEMMA } from '@/store/actions';
+import LexicalCategoryInput from '@/components/LexicalCategoryInput.vue';
+import {
+	UPDATE_LEMMA,
+	UPDATE_LEXICAL_CATEGORY,
```

Comment:
I actually wonder if we truly need actions for these, are we planning these update procedures to be `async` in the near future? Otherwise, we can just cut the middleman and use the mutations directly (luckily this is not redux :D)

Reply:
I’d also be okay with committing mutations directly, I think… does anyone else have objections to that?

Reply:
Mutations done in #41.